src/NonStrippingCreditCard.php

```diff
@@ -119,8 +118,7 @@ public function setToken($value)
     }
 
     /**
-     * Gets the token receieved from Apple Pay payment sheet.
-     * Includes the country, zip code, expiration date and account holder name.
+     * Gets the payment token.
```

Comment:
This was wrong. The token referred to before was actually the paymentData in the Apple Pay token. I also believe the comment should be more general incase another gateway added uses a token.

Reply:
See DataFaker function [here](https://github.com/vimeo/omnipay-vindicia/pull/59/files#diff-0c8d8b8afd3c0bd21b159506e01e7daeR1202).

Reply:
Actually, after discussing offline, we're going to have an Apple Pay specific token called `applePayToken`,  since omnipay-common's `token` is typed as a string.

Reply:
so does this getToken function need to exist? looks unused

Reply:
Right, it is. I'll remove all it's uses in this repo.